src/clj/rems/poller/email.clj

```diff
@@ -90,7 +95,7 @@
 
 (defmethod event-to-emails-impl :application.event/commented [event application]
   (vec
-   (for [handler (get-in application [:workflow :handlers])]
+   (for [handler (get-in application [:application/workflow :workflow.dynamic/handlers])]
```

Comment:
doens't application come from `applications/get-application-state`? that still uses the old keys right?

Reply:
Currently `applications/get-application-state` returns a map with both old and new keys, so that I can refactor things to use the new keys in small steps. This place has now been migrated to use the new keys, so I'll change it to get the v2 state directly.

Reply:
great, thanks